src/exporters/warpten.rs

```diff
@@ -0,0 +1,366 @@
+use crate::exporters::*;
+use crate::sensors::{RecordGenerator, Sensor, Topology};
+use std::collections::HashMap;
+use std::thread;
+use std::time::Duration;
+use utils::get_scaphandre_version;
+
+pub struct Warp10Exporter {
+    topology: Topology,
+}
+
+impl Exporter for Warp10Exporter {
+    //info!("Starting Warp10 exporter");
+    fn run(&mut self, parameters: clap::ArgMatches) {
+        let host = parameters.value_of("host").unwrap();
+        let scheme = parameters.value_of("scheme").unwrap();
+        let port = parameters.value_of("port").unwrap();
+        let write_token = parameters.value_of("write-token").unwrap();
+        let step = parameters.value_of("step").unwrap();
+        let qemu = parameters.is_present("qemu");
+
+        loop {
+            match self.iteration(
+                host,
+                scheme,
+                port.parse::<u16>().unwrap(),
+                write_token,
+                qemu,
+            ) {
+                Ok(res) => println!("Result: {:?}", res),
+                Err(err) => error!("Failed ! {:?}", err),
+            }
+            thread::sleep(Duration::new(step.parse::<u64>().unwrap(), 0));
+        }
+    }
+
+    fn get_options() -> HashMap<String, super::ExporterOption> {
+        let mut options = HashMap::new();
+
+        options.insert(
+            String::from("host"),
+            ExporterOption {
+                default_value: Some(String::from("localhost")),
+                help: String::from("Warp10 host's FQDN or IP address to send data to"),
+                long: String::from("host"),
+                short: String::from("H"),
+                required: false,
+                takes_value: true,
+            },
+        );
+        options.insert(
```

Comment:
I've done it like that because another option to send data to warp10 is websockets. Because the uri for that would look like ws://host/api/v0...
I thought it would be cleaner to have a separate variable to manage the protocol. 🤔

Reply:
Make sens in that case.